### PR TITLE
💄 #531: add loading animations

### PR DESCRIPTION
**Pull request by samrichca**
### Consent UI
- Add loading...
  - spinner
  - placeholder
  - button states
  - dictionary

## PR Readiness Checklist

- [x] "Expected Outcome(s)" in ticket have been met
- [x] Ticket number included in PR title
- [x] Connected ticket to PR
- [x] Labels added to PR for service name (`consent-api`, `data-mapper`, etc...), type (`chore`, `documentation`, etc...), status (`draft`, `on-hold`, etc...) **if applicable**
- [x] Manual testing completed
- [x] Builds locally without errors or warnings
- [x] Tests are updated (if required) and passing
- [ ] PR feedback has been addressed **if applicable**
- [x] New environment variables added to `.env.schema` files, `README.md`
- [x] Added copyrights to new files

### CODE REVIEW

File: apps/consent-ui/src/components/views/Dashboard/index.tsx [hunk 1]
```diff
@@ -58,15 +59,9 @@ const DashboardComponent = async ({ currentLang }: { currentLang: ValidLanguage
...
 					<div className={styles['button-container']}>
 						{consentStatus == 'complete' ? (
-							<LocalizedLink
-								name={'home'}
-								linkLang={currentLang}
-								role="button"
-								variant="secondary"
-								color="blue"
-							>
+							<Button variant="secondary" color="blue">
```
Comment (xusandrew) [inline apps/consent-ui/src/components/views/Dashboard/index.tsx:62]: tested this button here:

Screenshot 2024-04-04 at 2 28 57 PM

its not centered :(((

Comment (xusandrew) [reply apps/consent-ui/src/components/views/Dashboard/index.tsx:62]: Its a little off with "Loading" as the text too

**Comment by xusandrew** [comment]
Q: What places would we use this "loading" state, and how do we know a component is loading? Is it just [loading, setLoading] = useState?

**Comment by samrichca** [comment]
@xusandrew we'd use the button for downloads or other things that take a moment. where we get the loading state depends on the operation, we'll see

File: apps/consent-ui/src/components/views/Dashboard/index.tsx (hunk 1 above)
Comment (samrichca) [reply apps/consent-ui/src/components/views/Dashboard/index.tsx:62]: is that the big button on the homepage? I'll take a look

this is how it should look

image

Comment (samrichca) [reply apps/consent-ui/src/components/views/Dashboard/index.tsx]: fixed 

image 
image

**Review by xusandrew — APPROVED** [review]

**Pull request merged**